bin/grid

```diff
@@ -72,7 +72,7 @@ install_samza() {
   else
     mkdir -p $DOWNLOAD_CACHE_DIR
     pushd $DOWNLOAD_CACHE_DIR
-    git clone git://git.apache.org/incubator-samza.git
```

Comment:
This repository stopped responding. However, the github repository is claimed to be mirror of apache's.